ros2/images.yaml.em

```diff
@@ -23,6 +23,8 @@ images:
             - python3-pip
             - python3-setuptools
             - python3-vcstool
+            - libtinyxml-dev
```

Comment:
`urdf` package cannot be build without `libtinyxml-dev`. Actually it's a ROS 1 package as you already know.

The error log is shown below.

~~~
[ 50%] Building CXX object urdf_parser/CMakeFiles/urdfdom_world.dir/src/world.cpp.o
[ 53%] Linking CXX static library ../../lib/libgtest.a
[ 53%] Built target gtest
[ 57%] Linking CXX shared library ../lib/liburdfdom_world.so
/usr/bin/ld: /root/ros2_ws/install/lib/libtinyxml.a(tinyxml.o): relocation R_X86_64_32 against `.rodata' can not be used when making a shared object; recompile 
with -fPIC
/root/ros2_ws/install/lib/libtinyxml.a: error adding symbols: Bad value
collect2: error: ld returned 1 exit status
urdf_parser/CMakeFiles/urdfdom_world.dir/build.make:200: recipe for target 'lib/liburdfdom_world.so.1.0' failed
make[2]: *** [lib/liburdfdom_world.so.1.0] Error 1
make[1]: *** [urdf_parser/CMakeFiles/urdfdom_world.dir/all] Error 2
CMakeFiles/Makefile2:136: recipe for target 'urdf_parser/CMakeFiles/urdfdom_world.dir/all' failed
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2

<== Command '. /root/ros2_ws/build/urdfdom/cmake__build.sh && /usr/bin/make -j4 -l4' failed in '/root/ros2_ws/build/urdfdom' with exit code '2'
<== Command '. /root/ros2_ws/build/urdfdom/cmake__build.sh && /usr/bin/make -j4 -l4' failed in '/root/ros2_ws/build/urdfdom' with exit code '2'
The command '/bin/sh -c src/ament/ament_tools/scripts/ament.py     build     --build-tests     --symlink-install' returned a non-zero code: 1
~~~